packages/turbo-types/src/types/config.ts

```diff
@@ -4,71 +4,105 @@ export interface Schema {
    *
    * The contents of these files will be included in the global hashing
    * algorithm and affect the hashes of all tasks.
    *
-   * This is useful for busting the cache based on .env files (not in Git),
-   * or any root level file that impacts package tasks (but are not represented
-   * in the traditional dependency graph
+   * This is useful for busting the cache based on:
    *
-   * (e.g. a root tsconfig.json, jest.config.js, .eslintrc, etc.)).
+   * - .env files (not in Git)
+   *
+   * - any root level file that impacts package tasks
+   * that are not represented in the traditional dependency graph
+   * (e.g. a root tsconfig.json, jest.config.js, .eslintrc, etc.)
+   *
+   * Documentation: https://turbo.build/repo/docs/reference/configuration#globaldependencies
    *
    * @default []
    */
   globalDependencies?: string[];
 
   /**
-   * A list of environment variables, (e.g. GITHUB_TOKEN),
-   * for implicit global hash dependencies.
+   * A list of environment variables for implicit global hash dependencies.
+   *
+   * The variables included in this list will affect all task hashes.
+   *
+   * Documentation: https://turbo.build/repo/docs/reference/configuration#globalenv
    *
    * @default []
    */
   globalEnv?: string[];
 
   /**
    * An object representing the task dependency graph of your project. turbo interprets
-   * these conventions to properly schedule, execute, and cache the outputs of tasks in
+   * these conventions to schedule, execute, and cache the outputs of tasks in
    * your project.
    *
+   * Documentation: https://turbo.build/repo/docs/reference/configuration#pipeline
+   *
    * @default {}
    */
   pipeline: {
     /**
-     * The name of a task that can be executed by turbo run. If turbo finds a workspace
+     * The name of a task that can be executed by turbo. If turbo finds a workspace
      * package with a package.json scripts object with a matching key, it will apply the
-     * pipeline task configuration to that npm script during execution. This allows you to
-     * use pipeline to set conventions across your entire Turborepo.
+     * pipeline task configuration to that npm script during execution.
      */
     [script: string]: Pipeline;
   };
   /**
-   * Configuration options that control how turbo interfaces with the remote Cache.
+   * Configuration options that control how turbo interfaces with the remote cache.
+   *
+   * Documentation: https://turbo.build/repo/docs/core-concepts/remote-caching
+   *
    * @default {}
    */
   remoteCache?: RemoteCache;
+
+  /**
+   * This key is only available in Workspace Configs
+   * and cannot be used in your root turbo.json.
+   *
+   * Tells turbo to extend your root `turbo.json`
+   * and overrides with the keys provided
+   * in your Workspace Configs.
+   *
+   * Currently, only the "//" value is allowed.
+   *
+   * @default ["//"]
+   */
+  extends?: string[];
```

Comment:
Let's separate this into a new PR for a couple reasons:
- we want to hold it till the 1.8 release
- I'm looking into setting this up so the existence of this key prevents `globalEnv`, et al. I think it's [technically possible](https://json-schema.org/draft/2020-12/json-schema-core.html#name-keywords-for-applying-subsche), but we need to figure out how to write the JSON, and then see if the TS interface can even generate it.